dbms/src/Flash/Mpp/EstablishCall.cpp

```diff
@@ -78,129 +112,94 @@ void EstablishCallData::initRpc()
     try
     {
         FAIL_POINT_TRIGGER_EXCEPTION(FailPoints::random_tunnel_init_rpc_failure_failpoint);
-        service->establishMPPConnectionSyncOrAsync(&ctx, &request, nullptr, this);
+
+        auto [status, err_msg] = service->establishMPPConnectionSyncOrAsync(&ctx, &request, nullptr, this);
```

Comment:
nit: suggest a better name for `establishMPPConnectionSyncOrAsync`?

Reply:
I have no idea.